front/client/components/ArticleContent.vue

```diff
@@ -78,15 +80,27 @@
         </transition>
       </div>
     </div>
+    <confirm-dialog 
+      v-model="isConfirmDialogOpen"
+      :confirmationText="'delete this item'"
+      :confirmAction="deleteArticleContent"
+      :itemId="itemUid"
+    />
+    <!-- Shit happens because passing function as a prop somehow calls this function -->
```

Comment:
I decided to write code this way, because it is universal confirmation dialog that can be used anywhere in the application, you just need to pass function and text as a prop.